src/Sylius/Component/Product/Generator/CartesianSetBuilder.php

```diff
@@ -9,22 +9,35 @@
  * file that was distributed with this source code.
  */
 
-namespace Sylius\Component\Product\SetBuilder;
+namespace Sylius\Component\Product\Generator;
 
 /**
  * Builds the Cartesian product set from one or more given sets.
  *
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  */
-final class CartesianSetBuilder implements SetBuilderInterface
+final class CartesianSetBuilder
```

Comment:
Why did you remove interface? I guess cartesian set can be generated different way, based on a chosen algorithm